catalog/core/catalog-core-commons/src/main/java/ddf/catalog/impl/filter/TemporalFilter.java

```diff
@@ -125,19 +125,29 @@ private void setDates(Date startDate, Date endDate) {
     }
 
     public Date getStartDate() {
-        return startDate;
+        if(startDate != null) {
```

Comment:
We would be well-served creating two static methods available in a common library for encapsulating all this boilerplate get/set Date logic.